app/src/main/java/com/orange/ods/app/ui/components/buttons/icons/ButtonsIconToggle.kt

```diff
@@ -0,0 +1,87 @@
+/*
+ *
+ *  Copyright 2021 Orange
+ *
+ *  Use of this source code is governed by an MIT-style
+ *  license that can be found in the LICENSE file or at
+ *  https://opensource.org/licenses/MIT.
+ * /
+ */
+
+package com.orange.ods.app.ui.components.buttons.icons
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.verticalScroll
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.saveable.rememberSaveable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.dimensionResource
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import com.orange.ods.app.R
+import com.orange.ods.app.ui.components.buttons.InvertedBackgroundColumn
+import com.orange.ods.app.ui.utilities.composable.CodeImplementationColumn
+import com.orange.ods.app.ui.utilities.composable.CommonButtonTechnicalTextColumn
+import com.orange.ods.app.ui.utilities.composable.TechnicalText
+import com.orange.ods.compose.component.OdsComponent
+import com.orange.ods.compose.component.button.OdsIconToggleButton
+
+@Composable
+fun ButtonsIconToggle(customizationState: ButtonIconCustomizationState) {
+    val buttonCheckedState = rememberSaveable { mutableStateOf(false) }
```

Comment:
I don't know what is the correct behavior but I did that because of the code implementation section. The code changes according the customization, if we have 2 states, the code is not always corresponding to the state displayed. I don't know if I'm clear. But I'm not sure about the behavior, do you think this is better to have 2 states?

Reply:
It's perfectly clear, and it's a good reason. :+1: